genai-perf/genai_perf/config/generate/perf_analyzer_config.py

```diff
@@ -293,6 +293,8 @@ def _add_protocol_args(self, config: ConfigCommand) -> List[str]:
                 protocol_args += ["--shape", "max_tokens:1", "--shape", "text_input:1"]
         elif config.endpoint.service_kind == "openai":
             protocol_args += ["-i", "http"]
+        elif config.endpoint.service_kind == "tensorrtllm_engine":
+            protocol_args += ["--service-kind", "triton_c_api", "--streaming"]
```

Comment:
Thanks for fixing this. I noticed this bug a week or so ago but forgot to file a ticket.